src/main/java/org/gaul/s3proxy/Main.java

```diff
@@ -235,6 +246,15 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static PrintStream createLoggerErrorPrintStream() {
+        return new PrintStream(new LogOutputStream() {
+            @Override
+            protected void processLine(String s, int i) {
```

Comment:
Instead of bringing in some unnecessary dependency, can't you just override `write`?  Or fix this upstream in args4j?